tests/ut/test_base.py

```diff
@@ -231,6 +231,93 @@ def test_alt_build_key(self):
         cache = BaseCache(key_builder=lambda key, namespace: 'x')
         assert cache.build_key(pytest.KEY, 'namespace') == 'x'
 
+    @pytest.fixture
+    def set_test_ttl(self, base_cache):
+        base_cache.ttl = 10
+        yield
+        base_cache.ttl = None
```

Comment:
You don't need to reset the ttl to `None` every time. The `base_cache` fixture is per test so this fixture is not needed, just set the `ttl` of `base_cache` directly in the test whenever you need it

Reply:
Done